### PR TITLE
Use freddy in RabbitMQ adapter

### PR DESCRIPTION
**Pull request by SteamRiC**

### CODE REVIEW

File: lib/logasm.rb [hunk 1]
```diff
@@ -44,7 +41,7 @@ def log(level, *args)
     data = parse_log_data(*args)
 
     @adapters.each do |adapter|
-      adapter.log(level, data)
+      adapter.log(level, data.clone)
```
Comment (indrekj) [inline lib/logasm.rb]: why clone?

Comment (SteamRiC) [inline lib/logasm.rb]: stdout adapter deletes message from this hash and it will not be there for other adapters

Comment (indrekj) [inline lib/logasm.rb]: then it is time to change stdout adapter. mutating data = evil

**Comment by indrekj** [comment]
+1

**Pull request merged**